specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -389,6 +389,40 @@
           "type": "object"
         }
       }
+    },
+    "StorageBlobInventoryPolicyCompletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Storage.BlobInventoryPolicyCompleted event.",
+      "type": "object",
+      "properties": {
+        "scheduleDateTime": {
+          "description": "The time at which inventory policy was scheduled.",
+          "type": "string"
```

Comment:
Should it be a date-time? Does the format comply to ISO8601?
https://github.com/Azure/azure-rest-api-specs/blob/8fc90369b001f19f8ad5fd70da41c0025dd03cc8/specification/eventgrid/data-plane/Microsoft.EventHub/stable/2018-01-01/EventHub.json#L49

Reply:
Format do comply with ISO. We are converting date into ISO string ("%4d-%02d-%02dT%02d:%02d:%02dZ") and then adding it in data.

But not sure, if changing type have any other implication? I was following StorageLifecyclePolicyCompletedEventData which also have scheduleTime in same format but there type is string.

Reply:
Some of the language code generators take the `format` property into account when creating the model classes and will generate the property as a `DateTimeOffset` (for C#, other languages use different types) which can be more pleasant for consumers.

Adding `format` to the other events would be a breaking change, since the type of the property changes.  For new events like this, you could specify a format.

Reply:
Added